.github/workflows/runners.yaml

```
@@ -0,0 +1,89 @@
+name: Runners Deployment
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types: 
+      - completed
+      - requested
+
+env:
+  PAT: ${{ secrets.PAT }}
+  GOOGLE_PROJECT: ${{ secrets.GOOGLE_PROJECT }}
+  PULUMI_CONFIG_PASSPHRASE: ${{ secrets.PULUMI_CONFIG_PASSPHRASE }}
+  GOOGLE_CREDENTIALS: ${{ secrets.GCE_SA_KEY }}
+  GOOGLE_REGION: ${{ secrets.GOOGLE_REGION }}
+  PULUMI_STACK_NAME: ilgpu_${{ github.event.workflow_run.id }}
+  GS_BUCKET: ${{ secrets.GS_BUCKET }}
+  GCE_SA_EMAIL: ${{ secrets.GCE_SA_EMAIL }}
+
+jobs:
+  is-fork:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if repo is fork
+        id: is-fork
+        run: echo "::set-output name=fork::$(gh api repos/${{ github.repository }} | jq .fork)"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      fork: ${{ steps.is-fork.outputs.fork }}
+  deploy-runners:
+    runs-on: ubuntu-latest
+    needs: is-fork
+    if: ${{ github.event.action == 'requested' && needs.is-fork.outputs.fork == 'false' }}
+    environment: cuda-test-infra
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: ilgpu
+
+      - name: Checkout Pulumi Ephemeral Github Runner project
+        uses: actions/checkout@v2
+        with:
+          repository: pavlovic-ivan/ephemeral-github-runner-gcp
+          ref: main
+          path: pulumi
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Execute Pulumi
+        run: |
+          cd pulumi
+          curl -fsSL https://get.pulumi.com | sh
+          npm install
+          pulumi login $GS_BUCKET
+          pulumi stack init -s $PULUMI_STACK_NAME
+          pulumi up --diff --config-file ../ilgpu/config.yaml -y
+
+  destroy-runners:
+    runs-on: ubuntu-latest
+    needs: is-fork
+    if: ${{ github.event.action == 'completed' && needs.is-fork.outputs.fork == 'false' }}
+    environment: cuda-test-infra
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: ilgpu
+
+      - name: Checkout Pulumi Ephemeral Github Runner project
+        uses: actions/checkout@v2
+        with:
+          repository: pavlovic-ivan/ephemeral-github-runner-gcp
+          ref: main
+          path: pulumi
+
+      - name: Execute Pulumi
+        run: |
+          cd pulumi
+          curl -fsSL https://get.pulumi.com | sh
+          npm install
+          pulumi login $GS_BUCKET
+          pulumi stack select $PULUMI_STACK_NAME
+          pulumi destroy --config-file artifacts/config.yaml -y
```

This is the wrong path the config file
```suggestion
          pulumi destroy --config-file ../ilgpu/config.yaml -y
```

---

Thanks for pointing this out, i was sure i've updated the path to the file!